packages/@react-native-windows/cli/src/runWindows/utils/build.ts

```diff
@@ -78,6 +78,31 @@ export function getAppSolutionFile(options: RunWindowsOptions, config: Config) {
   }
 }
 
+export function restorePackageConfigs(options: RunWindowsOptions) {
+  const pkgConfigs = new Array();
+
+  if (!options.sln) {
+    return;
+  }
+
+  findFiles('TODO:GET SLN DIR', 'packages.config', pkgConfigs);
+}
+
+const fs = require('fs').fs;
```

Comment:
>require [](http://example.com/codeflow?start=11&length=7)

(nit) can this be done as regular "import" at the top of the file?

Reply:
Can and will. This is still a placeholder for the actual implementation :)